src/main/ngapp/src/app/select-language/select-language.component.css

```diff
@@ -0,0 +1,10 @@
+@import "~bootstrap/dist/css/bootstrap.min.css";
```

Comment:
I believe this has been imported in the main css file, so might not be necessary here

Reply:
cool. will remove in next pr